server/src/utils/protocol/attributes/kvstore/kvstore.transfer.ts

```diff
@@ -0,0 +1,46 @@
+import { Address } from "@liftedinit/many-js";
+import { parseAddress } from "../../../cbor-parsers";
+import { Analyzer } from "../../analyzer";
+
+interface ArgumentT {
+  key: string;
+  owner: string;
+  new_owner?: string;
+}
+
+type ResultT = null;
+
+interface EventT {
+  key: string;
+  owner: string;
+  new_owner?: string;
+}
+
+export class KvstoreTransferAnalyzer extends Analyzer<
+  ArgumentT,
+  ResultT,
+  EventT
+> {
+  static method = "kvstore.transfer";
+  static eventType = [13, 0];
+
+  parseArgs(sender: Address, payload: Map<any, any>): ArgumentT {
+    return {
+      key: payload.get(0),
+      owner: (parseAddress(payload.get(1)) || sender).toString(),
+      new_owner: parseAddress(payload.get(2)).toString(),
```

Comment:
I don't think that's right, see https://github.com/many-protocol/specification/blob/main/attributes/network/cddl/7_kvstore_commands.cddl